api/v1alpha1/componentversion_types.go

```diff
@@ -61,6 +61,7 @@ type ComponentVersionSpec struct {
 // Repository specifies access details for the repository that contains OCM ComponentVersions.
 type Repository struct {
 	// URL specifies the URL of the OCI registry in which the ComponentVersion is stored.
+	// MUST NOT CONTAIN THE SCHEME.
```

Comment:
@phoban01 @souleb I'm thinking about enforcing this by a webhook regex rule.

Reply:
yes if we plan to enforce https, better to fail early.